### ISSUE TITLE
consistant label private / public key

### ISSUE THREAD

**Issue by sbounmy** [sbounmy/hongbao_bitcoin#24]
I think the label "private key" and "public key" should not be placed with SVG.

it can be draw by the AI with its style

It should be at top (1st line) : center preferred but some design could be right.

<img width="529" alt="image" src="https://github.com/user-attachments/assets/bae39f9d-5edf-4039-9440-88e0a38fe40d">

![CleanShot 2024-12-09 at 21 58 45 2@2x](https://github.com/user-attachments/assets/17947473-8b5d-40af-94a0-623e34292003)

**Comment by SalmaTalbi** [comment]
Donc faudra changer les designs c'est ça?
![bitcoin_bill_wedding_front](https://github.com/user-attachments/assets/022b81d6-46ec-4423-8bc8-3c31a56eab51)
![bitcoin_bill_wedding_back](https://github.com/user-attachments/assets/6e0a7182-c774-4749-bc78-c8b7dc7e8234)

**Comment by sbounmy** [comment]
@SalmaTalbi exactly, maybe you could do can try with uppercase PRIVATE KEY / PUBLIC KEY
